deploy/crds/policy.open-cluster-management.io_operatorpolicies.yaml

```
@@ -66,47 +74,74 @@ spec:
+                      - Keep
+                      - Delete
+                      - DeleteIfUnused
+                    - enum:
+                      - Keep
+                      - Delete
                     default: Keep
                     description: |-
-                      Specifies whether to delete any CustomResourceDefinitions associated with the operator;
-                      defaults to 'Keep' because deleting them should be done deliberately
-                    enum:
-                    - Keep
-                    - Delete
+                      Use the customResourceDefinitions parameter to specify whether to delete any
+                      CustomResourceDefinitions associated with the operator. The default value is "Keep", because
+                      deleting them should be done deliberately.
                     type: string
                   operatorGroups:
+                    allOf:
+                    - enum:
+                      - Keep
+                      - Delete
+                      - DeleteIfUnused
+                    - enum:
+                      - Keep
+                      - DeleteIfUnused
                     default: DeleteIfUnused
                     description: |-
-                      Specifies whether to delete the OperatorGroup; defaults to 'DeleteIfUnused' which
-                      will only delete the OperatorGroup if there is not another resource using it.
-                    enum:
-                    - Keep
-                    - DeleteIfUnused
+                      Use the operatorGroups parameter to specify whether to delete the OperatorGroup. The default
+                      value is "DeleteIfUnused", which will only delete the OperatorGroup if there is not another
+                      resource using it.
                     type: string
                   subscriptions:
+                    allOf:
+                    - enum:
+                      - Keep
+                      - Delete
+                      - DeleteIfUnused
+                    - enum:
+                      - Keep
+                      - Delete
                     default: Delete
-                    description: Specifies whether to delete the Subscription; defaults
-                      to 'Delete'
-                    enum:
-                    - Keep
-                    - Delete
+                    description: |-
+                      Use the subscriptions parameter to specify whether to delete the Subscription. The default
+                      value is "Delete".
                     type: string
                 type: object
               severity:
-                description: 'Severity : low, medium, high, or critical'
+                description: |-
+                  Severity is a user-defined severity for when an object is noncompliant with this configuration
+                  policy. The supported options are "low", "medium", "high", and "critical".
```

Quotation mark consistency
```suggestion
                description: |-
                  Severity is a user-defined severity for when an object is non-compliant with this configuration
                  policy. The supported options are "low", "medium", "high", and "critical".
```